### ISSUE TITLE
Ensure accurate metrics in ScienceMesh dashboard (users, volume)

### ISSUE THREAD

**Issue by labkode** [sciencemesh/sciencemesh#170]
Check with each partners that they are publishing real metrics

**Comment by Daniel-WWU-IT** [comment]
I don't think anyone knows how to do that right now, including me. I have advised many times to send around an email with instructions on updating Reva and its configuration, since so much has changed in the past months. This has to be done by someone with the necessary technical background on Reva, which isn't me...

**Comment by labkode** [comment]
The numbers are set in the vendor apps (owncloud/nextcloud):
https://developer.sciencemesh.io/docs/obsolete/obsolete-efss-apps/nextcloud/

So as far as we have Reva running and looks like the case as we have bogus numbers I don't see why we couldn't get those numbers updated.

Do you or @david-antos knows why the nextlcoud/ownclud integration is flagged as obsolete?
Did something change on the monitoring infra that we can't get the data from the prometheus exporter that is in Reva?

**Comment by david-antos** [comment]
I just reported what @Daniel-WWU-IT explained (hopefully didn't misrepresented it), could Daniel provide some details?

**Comment by Daniel-WWU-IT** [comment]
The way these apps try to register themselves with the mesh has become obsolete. We do not want anything like that anymore, and if you read the new docs on the new website, you'll see that such a behavior contradicts the way we (WP2) have envisoned the whole ingress process. A lot has changed in the site ingress and how data is managed, too. This is why we marked them as obsolete. They could still be used to expose metrics I guess, but other than that, they'd just clog our system with useless requests.

**Comment by labkode** [comment]
@Daniel-WWU-IT, okay for the registration I understand but the underlying way metrics are gathered has not changed right?
We still pull the Reva prometheus endpoint?

**Comment by Daniel-WWU-IT** [comment]
Sure, that's still the same system, nothing has changed there. I believe we do have a metrics provider where you can hack your numbers in, right? Not just the "dummy" one that generates random stuff...

**Comment by labkode** [comment]
@Daniel-WWU-IT exactly, so if the sysadmins change these values for the json driver they will be able to produce real numbers, not sure though people will be able to perform such modifications in so little time ...

**Comment by Daniel-WWU-IT** [comment]
@labkode Maybe, but nonetheless, people should be informed about this, telling them how to configure that driver (which I don't know how to do). And those who're still using the ownCloud app just need to be poked to enter the latest numbers.
